synapse/daemon.py

```diff
@@ -466,7 +466,8 @@ def _getTaskFiniMesg(self, task, valu):
                     await link.tx(('t2:yield', {'retn': None}))
 
                 except Exception as e:
-                    logger.exception('error during async generator task')
+                    if not isinstance(e, asyncio.CancelledError):
+                        logger.exception('error during async generator task')
```

Comment:
couldn't this also occur in the non-async generator case?

Reply:
Probably? I didn't see that in the log. I'll reorg that code to use the same exception handler.